.github/workflows/ci-e2e-badger.yaml

```diff
@@ -1,15 +1,11 @@
 name: CIT Badger
 
 on:
-  push:
-    branches: [main]
-
-  pull_request:
-    branches: [main]
+  workflow_call:
 
 concurrency:
-  group: ${{ github.workflow }}-${{ (github.event.pull_request && github.event.pull_request.number) || github.ref || github.run_id }}
-  cancel-in-progress: true
+  group: cit-badger-${{ github.run_id }}
```

Comment:
run_id is unique, so this setting becomes useless. What we wanted is to cancel jobs in a PR if a new commit is pushed to the same PR.